crates/pathfinder/src/cairo/ext_py/ser.rs

```diff
@@ -57,6 +57,7 @@ impl From<crate::core::Chain> for UsedChain {
             crate::core::Chain::Testnet => UsedChain::Goerli,
             crate::core::Chain::Testnet2 => UsedChain::Goerli,
             crate::core::Chain::Integration => UsedChain::Goerli,
+            crate::core::Chain::Custom => UsedChain::Goerli,
```

Comment:
Instead of passing the chain we should just pass the chain ID to the python worker as that's the actual value that's needed there.

Reply:
Agreed but I just fell in with what we had already

Reply:
This is OK for now I think but once the chain ID will be configurable for custom chains we'll be forced to change this.